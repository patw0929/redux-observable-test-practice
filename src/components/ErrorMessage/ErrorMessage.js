import React from 'react';

const ErrorMessage = ({ text }) => (
  <div className="alert alert-danger" role="alert">
    <strong>Oops!</strong> {text}
  </div>
);

export default ErrorMessage;
