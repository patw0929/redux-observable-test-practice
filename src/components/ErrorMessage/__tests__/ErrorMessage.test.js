import React from 'react';
import { shallow } from 'enzyme';
import ErrorMessage from '../ErrorMessage';

describe('ErrorMessage', () => {
  it('renders correct as snapshot', () => {
    const subject = shallow(<ErrorMessage text="Network error!" />);

    expect(subject.html()).toMatchSnapshot();
  });
});
