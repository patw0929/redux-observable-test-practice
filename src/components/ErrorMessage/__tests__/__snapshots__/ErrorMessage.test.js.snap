// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders correct as snapshot 1`] = `"<div class=\\"alert alert-danger\\" role=\\"alert\\"><strong>Oops!</strong> Network error!</div>"`;
